src/utils/themes-manager.js

```diff
@@ -61,7 +91,9 @@ export class ThemesManager {
 
     for (const color in config.colors) {
       if (ACCENT_AND_ACKNOWLEDGEMENT_COLORS.includes(color)) {
-        this.setAccentOrAcknowledgementColor(color, config.colors[color]);
+        const colorValue = config.colors[color];
+        this.setAccentOrAcknowledgementColor(`${color}-${textContrastSuffix}`, this.makeTextContrastColor(colorValue));
+        this.setAccentOrAcknowledgementColor(color, colorValue);
```

Comment:
why is this called twice ?

Reply:
targets are different
1st call for text-color
2nd call for brand color